### DISCUSSION TITLE
to all of the idiots making "block lists" or "highlighters" for RMS supporters because you're butthurt

### DISCUSSION THREAD

**Discussion by RAYSONCOLLINS237** [rms-support-letter/rms-support-letter.github.io#4229] [General]
i gotta say you must be REALLY butthurt if you make blocklists of us

**Comment by AnimusPEXUS** [comment]
anti-RMS are useless for humanity - so why should we suffer form them blocking anything? it's like if we would suffer from pigeons or snails blocking us to speak to them. 

Imo, we better to concentrate on improving our beloved GNU project, as simple-minded ballast is finally gone.

**Reply by JosephTLyons** [reply]
"OuR BeLovEd GnU ProjeCt!!1!"

**Reply by clort81** [reply]
Do you not understand loving something because it's good, JosephTLyons?

**Reply by AnimusPEXUS** [reply]
> "OuR BeLovEd GnU ProjeCt!!1!"

just take your BLM/rainbowy crowd and go away silently.

**Reply by JosephTLyons** [reply]
> > "OuR BeLovEd GnU ProjeCt!!1!"
> 
> 
> 
> just take your BLM/rainbowy crowd and go away silently.

What a sad excuse for an insult.  Was this something that is supposed to hurt my feelings?  Lol.

**Reply by AnimusPEXUS** [reply]
> > > "OuR BeLovEd GnU ProjeCt!!1!"
> > 
> > 
> > just take your BLM/rainbowy crowd and go away silently.
> 
> What a sad excuse for an insult. Was this something that is supposed to hurt my feelings? Lol.

insult? hurt feelings? dude, you're just think too mutch of yourself. I just said you to go away: you are not needed.

**Reply by JosephTLyons** [reply]
> > > > "OuR BeLovEd GnU ProjeCt!!1!"
> 
> > > 
> 
> > > 
> 
> > > just take your BLM/rainbowy crowd and go away silently.
> 
> > 
> 
> > What a sad excuse for an insult. Was this something that is supposed to hurt my feelings? Lol.
> 
> 
> 
> insult? hurt feelings? dude, you're just think too mutch of yourself. I just said you to go away: you are not needed.

Prove that you *are* needed.  Rest assured, your beloved GNU project will be just fine without you, and without RMS.  Have a swell day, sir.

**Reply by AnimusPEXUS** [reply]
> > > > > "OuR BeLovEd GnU ProjeCt!!1!"
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > just take your BLM/rainbowy crowd and go away silently.
> > 
> > 
> > > 
> > 
> > 
> > > What a sad excuse for an insult. Was this something that is supposed to hurt my feelings? Lol.
> > 
> > 
> > insult? hurt feelings? dude, you're just think too mutch of yourself. I just said you to go away: you are not needed.
> 
> Prove that you _are_ needed. Rest assured, your beloved GNU project will be just fine without you, and without RMS. Have a swell day, sir.

lol, did anyone asked your opinion? just fork it and rip off. don't forget to change it's name.

**Reply by JosephTLyons** [reply]
> > > > > > "OuR BeLovEd GnU ProjeCt!!1!"
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > just take your BLM/rainbowy crowd and go away silently.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > What a sad excuse for an insult. Was this something that is supposed to hurt my feelings? Lol.
> 
> > > 
> 
> > > 
> 
> > > insult? hurt feelings? dude, you're just think too mutch of yourself. I just said you to go away: you are not needed.
> 
> > 
> 
> > Prove that you _are_ needed. Rest assured, your beloved GNU project will be just fine without you, and without RMS. Have a swell day, sir.
> 
> 
> 
> lol, did anyone asked your opinion? just fork it and rip off. don't forget to change it's name.

Luckily, I don't need anyone's permission to voice my opinion.  You have no status or authority in the GNU community, so, expect others to disregard your weird demands as well.
L

**Reply by AnimusPEXUS** [reply]
> > > > > > > "OuR BeLovEd GnU ProjeCt!!1!"
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > > just take your BLM/rainbowy crowd and go away silently.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > What a sad excuse for an insult. Was this something that is supposed to hurt my feelings? Lol.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > insult? hurt feelings? dude, you're just think too mutch of yourself. I just said you to go away: you are not needed.
> > 
> > 
> > > 
> > 
> > 
> > > Prove that you _are_ needed. Rest assured, your beloved GNU project will be just fine without you, and without RMS. Have a swell day, sir.
> > 
> > 
> > lol, did anyone asked your opinion? just fork it and rip off. don't forget to change it's name.
> 
> Luckily, I don't need anyone's permission to voice my opinion. You have no status or authority in the GNU community, so, expect others to disregard your weird demands as well.
> L

I think your just maniacally obsessed. Go see some treatment, clown.

**Reply by JosephTLyons** [reply]
> > > > > > > > "OuR BeLovEd GnU ProjeCt!!1!"
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > > just take your BLM/rainbowy crowd and go away silently.
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > What a sad excuse for an insult. Was this something that is supposed to hurt my feelings? Lol.
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > insult? hurt feelings? dude, you're just think too mutch of yourself. I just said you to go away: you are not needed.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > Prove that you _are_ needed. Rest assured, your beloved GNU project will be just fine without you, and without RMS. Have a swell day, sir.
> 
> > > 
> 
> > > 
> 
> > > lol, did anyone asked your opinion? just fork it and rip off. don't forget to change it's name.
> 
> > 
> 
> > Luckily, I don't need anyone's permission to voice my opinion. You have no status or authority in the GNU community, so, expect others to disregard your weird demands as well.
> 
> > L
> 
> 
> 
> I think your just maniacally obsessed...

It's weird to be called "maniacally obsessed" by someone who is actually obsessed with RMS.

**Reply by AnimusPEXUS** [reply]
> > > > > > > > > "OuR BeLovEd GnU ProjeCt!!1!"
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > > > > just take your BLM/rainbowy crowd and go away silently.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > > > What a sad excuse for an insult. Was this something that is supposed to hurt my feelings? Lol.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > > insult? hurt feelings? dude, you're just think too mutch of yourself. I just said you to go away: you are not needed.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > Prove that you _are_ needed. Rest assured, your beloved GNU project will be just fine without you, and without RMS. Have a swell day, sir.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > lol, did anyone asked your opinion? just fork it and rip off. don't forget to change it's name.
> > 
> > 
> > > 
> > 
> > 
> > > Luckily, I don't need anyone's permission to voice my opinion. You have no status or authority in the GNU community, so, expect others to disregard your weird demands as well.
> > 
> > 
> > > L
> > 
> > 
> > I think your just maniacally obsessed...
> 
> It's weird to be called "maniacally obsessed" by someone who is actually obsessed with RMS.

I'm not obsessed with Stallman, I'm obsessed with My freedoms. and I'm thankful to Stallaman, because it is he who started all this freedom for me. For me, he is just like father, in some sense. I'm living and breathing with free software for 20 years now, and for me, freedoms are more important than image of free software, which is "ruined" by RMS'es personal actions. this is not the case of marketing for me.

**Reply by weimzh** [reply]
> just take your BLM/rainbowy crowd and go away silently.

Just a note: RMS constantly supports BLM and LGBT rights (aka. rainbowy) in his personal website.

https://www.google.com/search?q=black+live+matter+site%3Astallman.org
https://www.google.com/search?q=transgender+site%3Astallman.org

Just add some facts.  I'm not interested in any arguments.

**Reply by AnimusPEXUS** [reply]
RMS is controversial person, who fights for personal privacy yet leaking Biden's ass. All this situation and more precise evaluation of he's doings, led me to new opinion: I'm grateful to him for starting and promoting free software, as I consider it a driver for science and progress, however, I also realized what he didn't done enough to protect GCC and GNU from raiding.

From all this mess, which is created around him, It is unclear now, which forces overthrown him.

Also, it is clear now, what Stallman really not good enough for particular project future, as now he more like passive speaker, rather than active leader.

Also, it is unclear, what will happen now to GCC, GNU and free software movement, as there are not seen person who can lead them by their original ideology and GCC looks more like RH property now.

**Comment by pda0** [comment]
We still can use those highlighters to find friends btw... 😉

**Comment by franpoli** [comment]
Sooner or later, with this spirit of exclusion, they will end up living in isolation.
